<ui version="4.0" >
 <class>Form</class>
 <widget class="QWidget" name="Form" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>351</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <widget class="QPushButton" name="_setDisplayButton" >
      <property name="text" >
       <string>setDisplay()</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="_display" >
      <property name="readOnly" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>40</y>
     <width>351</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <widget class="QPushButton" name="_createdWindowButton" >
      <property name="text" >
       <string>createWindow()</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="_createdWindow" >
      <property name="readOnly" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>100</y>
     <width>351</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <widget class="QPushButton" name="_setTitleTextButton" >
      <property name="text" >
       <string>setWindowTitleText()</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="_windowToSetTitleText" />
    </item>
    <item>
     <widget class="QLineEdit" name="_titleTextToSet" >
      <property name="text" >
       <string>some text</string>
      </property>
      <property name="readOnly" >
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>70</y>
     <width>351</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <widget class="QPushButton" name="_showWindowButton" >
      <property name="text" >
       <string>showWindow()</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="_windowToShow" >
      <property name="readOnly" >
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_4" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>130</y>
     <width>351</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <widget class="QPushButton" name="_getTitleTextButton" >
      <property name="text" >
       <string>getTitleText()</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="_windowToGetTitleText" />
    </item>
    <item>
     <widget class="QLineEdit" name="_titleText" >
      <property name="readOnly" >
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_5" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>180</y>
     <width>351</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <widget class="QPushButton" name="_hideWindowButton" >
      <property name="text" >
       <string>hideWindow()</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="_windowToHide" >
      <property name="readOnly" >
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_6" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>210</y>
     <width>351</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <widget class="QPushButton" name="_focusWindowButton" >
      <property name="text" >
       <string>focusWindow()</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="_windowToFocus" >
      <property name="readOnly" >
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_7" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>240</y>
     <width>351</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <widget class="QPushButton" name="_bringWindowOnTopButton" >
      <property name="text" >
       <string>bringWindowOnTop()</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="_windowToBring" >
      <property name="readOnly" >
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_8" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>270</y>
     <width>351</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <widget class="QPushButton" name="_sendWindowToBackButton" >
      <property name="text" >
       <string>sendWindowToBack()</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="_windowToSend" >
      <property name="readOnly" >
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
